src/main.cpp

```diff
@@ -0,0 +1,84 @@
+#include <clang-c/Index.h>
+#include <iostream>
+
+std::string convert(const CXString& s){
+   std::string result = clang_getCString(s);
+   clang_disposeString(s);
+   return result;
+}
+
+std::string get_cursor_kind(CXCursor cursor){
```

Comment:
You don't need to convert the cursor kind to strings, just use the enum instead.
https://clang.llvm.org/doxygen/group__CINDEX.html#gaaccc432245b4cd9f2d470913f9ef0013